fdbserver/workloads/AutomaticIdempotencyWorkload.actor.cpp

```diff
@@ -46,14 +46,19 @@ struct AutomaticIdempotencyWorkload : TestWorkload {
 	static constexpr auto NAME = "AutomaticIdempotencyCorrectness";
 	int64_t numTransactions;
 	Key keyPrefix;
+	int64_t minMinAgeSeconds;
 	double automaticPercentage;
+	constexpr static double slop = 2.0;
+	double pollingInterval;
 
 	bool ok = true;
 
 	AutomaticIdempotencyWorkload(WorkloadContext const& wcx) : TestWorkload(wcx) {
 		numTransactions = getOption(options, "numTransactions"_sr, 2500);
 		keyPrefix = KeyRef(getOption(options, "keyPrefix"_sr, "/autoIdempotency/"_sr));
+		minMinAgeSeconds = getOption(options, "minMinAgeSeconds"_sr, 15);
```

Comment:
[Optional] It might make sense to add values for `minMinAgeSeconds` and `pollingInterval` in the toml file as well so that the configuration of the workload is immediately clear to a reader.